docs/upgrades/unreleased.md

```diff
@@ -129,6 +129,21 @@
 
 If you plan to use the Angular SDK with XMCloud, you will need to perform next steps:
 
+* Add dependencies for CloudSDK libraries in `package.json`:
+    ```
+        "@sitecore-cloudsdk/core": "^0.4.0",
+        "@sitecore-cloudsdk/events": "^0.4.0",
+    ```
+* Add an extra export in `server.exports.ts`:
```

Comment:
We already have a section below that asks to add this file, can you merge this change under existing section?